enhancements/cert-manager/cert-manager-network-policies.md

```
@@ -0,0 +1,413 @@
+    kind: NetworkPolicy
+    metadata:
+      name: deny-all-traffic
+      namespace: cert-manager-operator
+    spec:
+      podSelector: {}
+      policyTypes:
+      - Ingress
+      - Egress
+    ```
+
+2.  **Allow Operator Egress to API Server & Ingress for Metrics:**
```

markdownlint-cli2 (0.17.2)

Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)